cmd/sts-handlers.go

```diff
@@ -45,82 +44,104 @@ func registerSTSRouter(router *mux.Router) {
 	// AssumeRoleWithClientGrants
 	stsRouter.Methods("POST").HandlerFunc(httpTraceAll(sts.AssumeRoleWithClientGrants)).
 		Queries("Action", "AssumeRoleWithClientGrants").
+		Queries("Version", stsAPIVersion).
 		Queries("Token", "{Token:.*}")
 
-}
+	// AssumeRoleWithWebIdentity
+	stsRouter.Methods("POST").HandlerFunc(httpTraceAll(sts.AssumeRoleWithWebIdentity)).
```

Comment:
Probably, stsRouter should avoid logging here, since this can leak some information

Reply:
There is no information leak @vadmeste - one can never access or validate anything here. The request itself is anonymous so I don't see removing this saves us from anything.